<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 Gunnar von Heijne semminar | Structural Bioinformatics</title>
  <meta name="description" content="This is a book for the notes and activities of the structural bioinformatics module of the Bioinformatics and Statistics course." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 Gunnar von Heijne semminar | Structural Bioinformatics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a book for the notes and activities of the structural bioinformatics module of the Bioinformatics and Statistics course." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 Gunnar von Heijne semminar | Structural Bioinformatics" />
  
  <meta name="twitter:description" content="This is a book for the notes and activities of the structural bioinformatics module of the Bioinformatics and Statistics course." />
  

<meta name="author" content="Alfredo Varela, Paulina Rosales" />


<meta name="date" content="2021-03-12" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="secuencias.html"/>
<link rel="next" href="cotranslational-protein-folding-the-when-and-why-of-che.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Bioinformática y Estadística II-III</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Prerequisites</a></li>
<li class="chapter" data-level="2" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html"><i class="fa fa-check"></i><b>2</b> 3 Cs of covalent bondas</a><ul>
<li class="chapter" data-level="2.1" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#shape-of-biological-molecules"><i class="fa fa-check"></i><b>2.1</b> Shape of biological molecules</a></li>
<li class="chapter" data-level="2.2" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#recognition-and-function-of-biological-molecules"><i class="fa fa-check"></i><b>2.2</b> Recognition and function of biological molecules</a></li>
<li class="chapter" data-level="2.3" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#macromolecular-building-blocks"><i class="fa fa-check"></i><b>2.3</b> Macromolecular Building Blocks</a></li>
<li class="chapter" data-level="2.4" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#video-dna"><i class="fa fa-check"></i><b>2.4</b> video DNA</a></li>
<li class="chapter" data-level="2.5" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#evolutivamente"><i class="fa fa-check"></i><b>2.5</b> Evolutivamente</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="xds-inp.html"><a href="xds-inp.html"><i class="fa fa-check"></i><b>3</b> XDS.INP</a><ul>
<li class="chapter" data-level="3.1" data-path="xds-inp.html"><a href="xds-inp.html#lyzosime"><i class="fa fa-check"></i><b>3.1</b> Lyzosime</a></li>
<li class="chapter" data-level="3.2" data-path="xds-inp.html"><a href="xds-inp.html#life-is-the-result-of-the-chemical-activity-of-proteins"><i class="fa fa-check"></i><b>3.2</b> Life is the result of the chemical activity of proteins</a><ul>
<li class="chapter" data-level="3.2.1" data-path="xds-inp.html"><a href="xds-inp.html#determinación-de-las-bases-con-molecular-replacement"><i class="fa fa-check"></i><b>3.2.1</b> Determinación de las bases con molecular replacement</a></li>
<li class="chapter" data-level="3.2.2" data-path="xds-inp.html"><a href="xds-inp.html#molecular-replacement"><i class="fa fa-check"></i><b>3.2.2</b> Molecular replacement</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="xds-inp.html"><a href="xds-inp.html#alpha-trace-c"><i class="fa fa-check"></i><b>3.3</b> alpha trace C</a><ul>
<li class="chapter" data-level="3.3.1" data-path="xds-inp.html"><a href="xds-inp.html#primary-structure-peptide-bond"><i class="fa fa-check"></i><b>3.3.1</b> Primary structure peptide bond</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="xds-inp.html"><a href="xds-inp.html#preguntas-a-responder"><i class="fa fa-check"></i><b>3.4</b> Preguntas a responder</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="literature.html"><a href="literature.html"><i class="fa fa-check"></i><b>4</b> Literature</a></li>
<li class="chapter" data-level="5" data-path="pymol.html"><a href="pymol.html"><i class="fa fa-check"></i><b>5</b> Pymol</a><ul>
<li class="chapter" data-level="5.1" data-path="pymol.html"><a href="pymol.html#uniprot"><i class="fa fa-check"></i><b>5.1</b> Uniprot</a></li>
<li class="chapter" data-level="5.2" data-path="pymol.html"><a href="pymol.html#subir-tu-proteína-a-pdb"><i class="fa fa-check"></i><b>5.2</b> subir tu proteína a PDB</a></li>
<li class="chapter" data-level="5.3" data-path="pymol.html"><a href="pymol.html#estructuras"><i class="fa fa-check"></i><b>5.3</b> Estructuras</a></li>
<li class="chapter" data-level="5.4" data-path="pymol.html"><a href="pymol.html#pymol-by-schrodinger"><i class="fa fa-check"></i><b>5.4</b> Pymol by Schrodinger</a><ul>
<li class="chapter" data-level="5.4.1" data-path="pymol.html"><a href="pymol.html#pdb-structrue-file"><i class="fa fa-check"></i><b>5.4.1</b> PDB structrue file</a></li>
<li class="chapter" data-level="5.4.2" data-path="pymol.html"><a href="pymol.html#interactive-command-line"><i class="fa fa-check"></i><b>5.4.2</b> Interactive command line</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="pymol.html"><a href="pymol.html#sars-cov-2"><i class="fa fa-check"></i><b>5.5</b> SARS-Cov-2</a><ul>
<li class="chapter" data-level="5.5.1" data-path="pymol.html"><a href="pymol.html#s-protein-spike"><i class="fa fa-check"></i><b>5.5.1</b> S protein (Spike)</a></li>
<li class="chapter" data-level="5.5.2" data-path="pymol.html"><a href="pymol.html#paper"><i class="fa fa-check"></i><b>5.5.2</b> Paper</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="pymol.html"><a href="pymol.html#figure-scripting-by-example-interactive-automation-with-hotkeys-pymolrc-startup-script"><i class="fa fa-check"></i><b>5.6</b> Figure scripting by example Interactive Automation with hotkeys pymolrc startup script</a></li>
<li class="chapter" data-level="5.7" data-path="pymol.html"><a href="pymol.html#access-and-modify-atom-data-with-iterate-and-alter-write-pymol-extensions"><i class="fa fa-check"></i><b>5.7</b> Access and modify atom data with iterate and alter write PyMOL extensions</a></li>
<li class="chapter" data-level="5.8" data-path="pymol.html"><a href="pymol.html#anaconda-cloud-with-pymol-2.0"><i class="fa fa-check"></i><b>5.8</b> Anaconda Cloud with PyMOL 2.0</a></li>
<li class="chapter" data-level="5.9" data-path="pymol.html"><a href="pymol.html#exercises"><i class="fa fa-check"></i><b>5.9</b> Exercises</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="secuencias.html"><a href="secuencias.html"><i class="fa fa-check"></i><b>6</b> Secuencias</a><ul>
<li class="chapter" data-level="6.1" data-path="secuencias.html"><a href="secuencias.html#the-protein-side-of-the-central-dogma-permanence-and-change-thermoynamic-theory-of-protein-folding"><i class="fa fa-check"></i><b>6.1</b> The protein side of the central dogma: Permanence and change thermoynamic theory of protein folding</a></li>
<li class="chapter" data-level="6.2" data-path="secuencias.html"><a href="secuencias.html#in-vitro-models"><i class="fa fa-check"></i><b>6.2</b> In vitro Models</a></li>
<li class="chapter" data-level="6.3" data-path="secuencias.html"><a href="secuencias.html#gibbs-free-energy-explained"><i class="fa fa-check"></i><b>6.3</b> Gibbs free Energy Explained</a></li>
<li class="chapter" data-level="6.4" data-path="secuencias.html"><a href="secuencias.html#protein-folding-kinetics"><i class="fa fa-check"></i><b>6.4</b> Protein Folding Kinetics</a></li>
<li class="chapter" data-level="6.5" data-path="secuencias.html"><a href="secuencias.html#free-energy-of-folding-for-taq"><i class="fa fa-check"></i><b>6.5</b> Free energy of folding for Taq</a></li>
<li class="chapter" data-level="6.6" data-path="secuencias.html"><a href="secuencias.html#thermodynamics"><i class="fa fa-check"></i><b>6.6</b> Thermodynamics</a></li>
<li class="chapter" data-level="6.7" data-path="secuencias.html"><a href="secuencias.html#phi-value-analysis"><i class="fa fa-check"></i><b>6.7</b> Phi Value analysis</a></li>
<li class="chapter" data-level="6.8" data-path="secuencias.html"><a href="secuencias.html#ejercicio-real"><i class="fa fa-check"></i><b>6.8</b> Ejercicio Real</a></li>
<li class="chapter" data-level="6.9" data-path="secuencias.html"><a href="secuencias.html#genealogy"><i class="fa fa-check"></i><b>6.9</b> Genealogy</a><ul>
<li class="chapter" data-level="6.9.1" data-path="secuencias.html"><a href="secuencias.html#comprehensive-databases"><i class="fa fa-check"></i><b>6.9.1</b> comprehensive databases</a></li>
<li class="chapter" data-level="6.9.2" data-path="secuencias.html"><a href="secuencias.html#other-databases"><i class="fa fa-check"></i><b>6.9.2</b> other databases</a></li>
</ul></li>
<li class="chapter" data-level="6.10" data-path="secuencias.html"><a href="secuencias.html#chemical-methods-and-thermal-denaturation-estimation-of-thermodynamic-stability-disordered-proteins"><i class="fa fa-check"></i><b>6.10</b> Chemical methods and thermal denaturation Estimation of thermodynamic stability Disordered proteins</a></li>
<li class="chapter" data-level="6.11" data-path="secuencias.html"><a href="secuencias.html#the-ribosome"><i class="fa fa-check"></i><b>6.11</b> The ribosome</a></li>
<li class="chapter" data-level="6.12" data-path="secuencias.html"><a href="secuencias.html#evolution-of-the-ribosome"><i class="fa fa-check"></i><b>6.12</b> Evolution of the ribosome</a></li>
<li class="chapter" data-level="6.13" data-path="secuencias.html"><a href="secuencias.html#interactive-exercises-protein-folding-and-the-ribosome."><i class="fa fa-check"></i><b>6.13</b> Interactive exercises protein folding and the ribosome.</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html"><i class="fa fa-check"></i><b>7</b> Gunnar von Heijne semminar</a><ul>
<li class="chapter" data-level="7.1" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#the-signal-sequence"><i class="fa fa-check"></i><b>7.1</b> The signal sequence</a></li>
<li class="chapter" data-level="7.2" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#can-someone-identify-such-sequences-with-bioinforatics-on-data-bases"><i class="fa fa-check"></i><b>7.2</b> Can someone identify such sequences with bioinforatics on data bases</a></li>
<li class="chapter" data-level="7.3" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#machine-learning-approaches"><i class="fa fa-check"></i><b>7.3</b> Machine learning approaches</a><ul>
<li class="chapter" data-level="7.3.1" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#you-need"><i class="fa fa-check"></i><b>7.3.1</b> You need</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#memebrane-proteins"><i class="fa fa-check"></i><b>7.4</b> Memebrane proteins</a></li>
<li class="chapter" data-level="7.5" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#topology-prediction"><i class="fa fa-check"></i><b>7.5</b> Topology prediction</a><ul>
<li class="chapter" data-level="7.5.1" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#toppred"><i class="fa fa-check"></i><b>7.5.1</b> TopPred</a></li>
<li class="chapter" data-level="7.5.2" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#tmhmm"><i class="fa fa-check"></i><b>7.5.2</b> TMHMM</a></li>
<li class="chapter" data-level="7.5.3" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#topcons"><i class="fa fa-check"></i><b>7.5.3</b> TOPCONS</a></li>
<li class="chapter" data-level="7.5.4" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#how-good-are-topology-predictors"><i class="fa fa-check"></i><b>7.5.4</b> How good are topology predictors</a></li>
<li class="chapter" data-level="7.5.5" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#d-structure-of-membrane-proteinns-from-genomic-sequencing"><i class="fa fa-check"></i><b>7.5.5</b> 3D structure of membrane proteinns from Genomic Sequencing</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html"><i class="fa fa-check"></i><b>8</b> Cotranslational protein folding: the when and why of che</a><ul>
<li class="chapter" data-level="8.1" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#defects-in-protein-folding-lead-to-human-disease"><i class="fa fa-check"></i><b>8.1</b> Defects in protein folding lead to human disease</a></li>
<li class="chapter" data-level="8.2" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#molecular-chaperone-socntrol-every-aspect-f-cellular-protein-homeostasis"><i class="fa fa-check"></i><b>8.2</b> Molecular chaperone socntrol every aspect f cellular protein homeostasis</a></li>
<li class="chapter" data-level="8.3" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#clip-chaperones-linked-to-protein-synthesis-folding-of-newly-made-proteins"><i class="fa fa-check"></i><b>8.3</b> CLIP chaperones linked to protein synthesis folding of newly made proteins</a></li>
<li class="chapter" data-level="8.4" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#co-translation-folding-the-vectorial-nature-of-translation-determines-the-folding-pathway"><i class="fa fa-check"></i><b>8.4</b> Co-translation folding: the vectorial nature of translation determines the folding pathway</a></li>
<li class="chapter" data-level="8.5" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#biogenesis-begins-at-the-ribosome"><i class="fa fa-check"></i><b>8.5</b> Biogenesis begins at the ribosome</a><ul>
<li class="chapter" data-level="8.5.1" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#interaction-of-nascent-chains-with-hsp70"><i class="fa fa-check"></i><b>8.5.1</b> interaction of nascent chains with HSP70</a></li>
</ul></li>
<li class="chapter" data-level="8.6" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#how-do-hsp70-and-tric-engage-their-cognate-substrates-co-translationally"><i class="fa fa-check"></i><b>8.6</b> How do Hsp70 and TRiC engage their cognate substrates co-translationally?</a></li>
<li class="chapter" data-level="8.7" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#what-detemines-binding-kinetics-of-elongation-rate"><i class="fa fa-check"></i><b>8.7</b> what detemines binding: kinetics of elongation rate?</a></li>
<li class="chapter" data-level="8.8" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#why-we-age"><i class="fa fa-check"></i><b>8.8</b> ## why we age</a></li>
<li class="chapter" data-level="8.9" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#practice"><i class="fa fa-check"></i><b>8.9</b> Practice</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="protein-design.html"><a href="protein-design.html"><i class="fa fa-check"></i><b>9</b> Protein design</a><ul>
<li class="chapter" data-level="9.1" data-path="protein-design.html"><a href="protein-design.html#power-of-evolution"><i class="fa fa-check"></i><b>9.1</b> Power of evolution</a></li>
<li class="chapter" data-level="9.2" data-path="protein-design.html"><a href="protein-design.html#native-structures-are-conformational-free-energy-minima"><i class="fa fa-check"></i><b>9.2</b> Native structures are conformational free energy minima</a></li>
<li class="chapter" data-level="9.3" data-path="protein-design.html"><a href="protein-design.html#why-bother"><i class="fa fa-check"></i><b>9.3</b> Why bother?</a></li>
<li class="chapter" data-level="9.4" data-path="protein-design.html"><a href="protein-design.html#protein-engineering-and-design-approaches"><i class="fa fa-check"></i><b>9.4</b> Protein engineering and design approaches</a><ul>
<li class="chapter" data-level="9.4.1" data-path="protein-design.html"><a href="protein-design.html#summary"><i class="fa fa-check"></i><b>9.4.1</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="protein-design.html"><a href="protein-design.html#fragment-based-design"><i class="fa fa-check"></i><b>9.5</b> Fragment-based design</a><ul>
<li class="chapter" data-level="9.5.1" data-path="protein-design.html"><a href="protein-design.html#how-do-all-folds-compare"><i class="fa fa-check"></i><b>9.5.1</b> How do all folds compare</a></li>
<li class="chapter" data-level="9.5.2" data-path="protein-design.html"><a href="protein-design.html#summary-iii"><i class="fa fa-check"></i><b>9.5.2</b> summary III</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Structural Bioinformatics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="gunnar-von-heijne-semminar" class="section level1">
<h1><span class="header-section-number">Chapter 7</span> Gunnar von Heijne semminar</h1>
<p>Signal peptides
- targeting information
- Chloroplasts targeting sequences
- mitocondrial targeting sequences
- peroxisome targeting sequences</p>
<div id="the-signal-sequence" class="section level2">
<h2><span class="header-section-number">7.1</span> The signal sequence</h2>
<ul>
<li>Extension in the n terminus that encodes targeting information</li>
<li>Common architecture
<ul>
<li>tripod structure
<ul>
<li>n-region: positively charged</li>
<li>h-region: hydrophobic
<ul>
<li>valines</li>
</ul></li>
<li>c-region: more polar, small residues in -1,-3
<ul>
<li>alanine, serine or glicine</li>
<li>protease acts</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="can-someone-identify-such-sequences-with-bioinforatics-on-data-bases" class="section level2">
<h2><span class="header-section-number">7.2</span> Can someone identify such sequences with bioinforatics on data bases</h2>
<blockquote>
<p>A new method for predicting signal sequence</p>
</blockquote>
<ul>
<li>using PSSM position specific scoring matrix</li>
<li>Exerything was allign to the -1
<ul>
<li>where smaller amino acids tend to be</li>
</ul></li>
<li>just counts and a frequency</li>
<li>nurmalizing the count by the expected count
<ul>
<li>taking the natural log
<ul>
<li>less than exped would be less than one so ol will be negative</li>
</ul></li>
<li>for every sequence you can get a score</li>
<li>signal sequence vs no signal sequence</li>
<li>real separation between cytosolic sequences</li>
<li>how can you predict the cleavage site?</li>
</ul></li>
</ul>
</div>
<div id="machine-learning-approaches" class="section level2">
<h2><span class="header-section-number">7.3</span> Machine learning approaches</h2>
<ul>
<li>SignalP with artificial neural networks</li>
<li>Input layer
<ul>
<li>where you read</li>
<li>depending on the aminoacid differen values</li>
</ul></li>
<li>1st hidden layer
<ul>
<li>input feeds it</li>
</ul></li>
<li>2nd hidden layer</li>
<li>output layer
<ul>
<li>high output is probably a signal sequence (yes)</li>
<li>not a signal sequence (no)</li>
</ul></li>
</ul>
<p>Imagine you are
- hydrophobic 1
- polar 0
- charge 0
1. is a hydrophobic</p>
<p>Wi= the first weight, each node is propagated with a weight which initialy is undecided</p>
<p>each conection between input layer and 1st hidden layer</p>
<p>x1 by w1 =a
x2 by w2 = b
x3 by w3 = c</p>
<p>Then you sum the values a`+b+c and you transform the result with a function such that if your sum is less than the threshold the output to the next one will be
- strong enough input makes a strong output to propagate to 2nd hidden layer
- How do you choose the values for the weight?
- random
- there is a procedure where you can train the weights, optimization with true negative and true positive
- true sequence peptides</p>
<ul>
<li>you want as few as possible hidden layers
<ul>
<li>easiest is to start with one</li>
</ul></li>
<li>feed forward networks
<ul>
<li>unidirectional</li>
</ul></li>
<li></li>
</ul>
<div id="you-need" class="section level3">
<h3><span class="header-section-number">7.3.1</span> You need</h3>
<ul>
<li>high-quality dataset of positive and negative examples</li>
<li>choose the architecture
<ul>
<li>can be optimized</li>
</ul></li>
<li>all internal parameter in the ANN artificial neural network</li>
<li>you take 20 of both and you train with 80 both and test 20 to test and then you can take another 20 to test and train with other 80 to make the pondered mean</li>
</ul>
<p>MMC detection will be 1 if your predictions are good
- CS precision
- number of cleaveage site predictions that are correct
- CS recall
- what the fraction where with the</p>
<p>## Protein sorting in a eukaryotic cell
- TargetP tries to predict where the signal comes from
- mitoP
- SignalP
- ChloroP
- TatP
- LipoP
<a href="www.cbs.dtu.dk">targetP</a></p>
<p>So the main goal of choosing a function is to</p>
</div>
</div>
<div id="memebrane-proteins" class="section level2">
<h2><span class="header-section-number">7.4</span> Memebrane proteins</h2>
<ul>
<li>Have been designed by nature to optimately locate between lipid bilayers</li>
<li>two main designs
<ul>
<li>helix bundle
<ul>
<li>alfa helices with mostly hydrophobic aminoacids</li>
</ul></li>
<li>beta barrel
<ul>
<li>antiparalel beta sheet</li>
<li>inside is polar</li>
<li>outside is hidophobic</li>
</ul></li>
</ul></li>
<li>Helix bundle membrane protein
<ul>
<li>topology diagram
<ul>
<li>where in the sequence</li>
<li>overall orientation</li>
<li>1 orientation in 99%</li>
<li>you want to preditct
<ul>
<li>where are transmembran helices</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<p>TM helices are
- 1 aminoacid apports 1.5 armstrong to the length of a helix
- helices are inserted independently
- helices 20- 30 residues long
- trp and tyr big aromatic residues that accumulate in the lipid head regions flanking the core
- they signal the end of the beta strands
- loops tend to be 10- 20 residues
- quite short
- if you compare compoistion between the loops facing
- inward the cell facing tendt to have 3 t o 4 more k and R positively charge aminoacids
- almost a universal truth</p>
</div>
<div id="topology-prediction" class="section level2">
<h2><span class="header-section-number">7.5</span> Topology prediction</h2>
<p>You said that this is a mature field, the current major discoveries are being done in 3D structure or 3D to 3D interacition?</p>
<ul>
<li>old field<br />
</li>
<li>TMHMM</li>
<li>HMMTOP</li>
</ul>
<ol style="list-style-type: decimal">
<li>hidrophobicity</li>
<li>positive bias</li>
<li>Machine learning</li>
</ol>
<div id="toppred" class="section level3">
<h3><span class="header-section-number">7.5.1</span> TopPred</h3>
<ul>
<li>hidophobity analysis with positive inside rule</li>
<li>hidrophobicity plot
<ul>
<li>widow of 20 residues
<ul>
<li>average of hidrophobicity
<ul>
<li>negative numbers to charge and polar residues</li>
</ul></li>
</ul></li>
</ul></li>
<li>hard to create the cut-off so this ended the methods</li>
<li>it added positive inside rule</li>
<li>one cut-off chosen so high that it must show hidrophobicity</li>
<li>a second cut that shows certainly they are not</li>
<li>it counts positive charges</li>
</ul>
</div>
<div id="tmhmm" class="section level3">
<h3><span class="header-section-number">7.5.2</span> TMHMM</h3>
<ul>
<li>you formulate a model
tail(out)-helix-inside loop-helix-outsideloop-helix</li>
<li>in each node you have a program
<ul>
<li>alignmente or a certain score?</li>
</ul></li>
<li>markov model</li>
</ul>
</div>
<div id="topcons" class="section level3">
<h3><span class="header-section-number">7.5.3</span> TOPCONS</h3>
<ul>
<li>web site</li>
<li>you submit your sequence</li>
</ul>
</div>
<div id="how-good-are-topology-predictors" class="section level3">
<h3><span class="header-section-number">7.5.4</span> How good are topology predictors</h3>
<ul>
<li>single sequence predictor 70-75%</li>
<li>mutli-sequence 80-85%</li>
<li>97%</li>
<li>bacteria are easier than eukaryotes</li>
</ul>
</div>
<div id="d-structure-of-membrane-proteinns-from-genomic-sequencing" class="section level3">
<h3><span class="header-section-number">7.5.5</span> 3D structure of membrane proteinns from Genomic Sequencing</h3>
<ul>
<li>alpha-fold two</li>
<li>huge protein sequence database
<ul>
<li>sure you ll find a homoloug for your protein</li>
</ul></li>
<li>co-evolving residues
<ul>
<li>you have multiple sequence alignment and you look for coinciden patterns of mutation</li>
<li>if i can identify this type of co-evolving pairs there is a high probability that if one mutates the other one must mutate in order to keep the structure</li>
</ul></li>
</ul>
<p>Can you see the same concept of cooevolution between interacting proteins? yes you can</p>
<p>coevolution and deep network
what s the difference between deep network and artificial neural networks</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="secuencias.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="cotranslational-protein-folding-the-when-and-why-of-che.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["BEII_III_Structural_Bioinfo.pdf", "BEII_III_Structural_Bioinfo.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
