<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Pymol | Structural Bioinformatics</title>
  <meta name="description" content="This is a book for the notes and activities of the structural bioinformatics module of the Bioinformatics and Statistics course." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Pymol | Structural Bioinformatics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a book for the notes and activities of the structural bioinformatics module of the Bioinformatics and Statistics course." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Pymol | Structural Bioinformatics" />
  
  <meta name="twitter:description" content="This is a book for the notes and activities of the structural bioinformatics module of the Bioinformatics and Statistics course." />
  

<meta name="author" content="Alfredo Varela, Paulina Rosales" />


<meta name="date" content="2021-03-10" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="literature.html"/>
<link rel="next" href="secuencias.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Bioinformática y Estadística II-III</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Prerequisites</a></li>
<li class="chapter" data-level="2" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html"><i class="fa fa-check"></i><b>2</b> 3 Cs of covalent bondas</a><ul>
<li class="chapter" data-level="2.1" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#shape-of-biological-molecules"><i class="fa fa-check"></i><b>2.1</b> Shape of biological molecules</a></li>
<li class="chapter" data-level="2.2" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#recognition-and-function-of-biological-molecules"><i class="fa fa-check"></i><b>2.2</b> Recognition and function of biological molecules</a></li>
<li class="chapter" data-level="2.3" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#macromolecular-building-blocks"><i class="fa fa-check"></i><b>2.3</b> Macromolecular Building Blocks</a></li>
<li class="chapter" data-level="2.4" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#video-dna"><i class="fa fa-check"></i><b>2.4</b> video DNA</a></li>
<li class="chapter" data-level="2.5" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#evolutivamente"><i class="fa fa-check"></i><b>2.5</b> Evolutivamente</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="xds-inp.html"><a href="xds-inp.html"><i class="fa fa-check"></i><b>3</b> XDS.INP</a><ul>
<li class="chapter" data-level="3.1" data-path="xds-inp.html"><a href="xds-inp.html#lyzosime"><i class="fa fa-check"></i><b>3.1</b> Lyzosime</a></li>
<li class="chapter" data-level="3.2" data-path="xds-inp.html"><a href="xds-inp.html#life-is-the-result-of-the-chemical-activity-of-proteins"><i class="fa fa-check"></i><b>3.2</b> Life is the result of the chemical activity of proteins</a><ul>
<li class="chapter" data-level="3.2.1" data-path="xds-inp.html"><a href="xds-inp.html#determinación-de-las-bases-con-molecular-replacement"><i class="fa fa-check"></i><b>3.2.1</b> Determinación de las bases con molecular replacement</a></li>
<li class="chapter" data-level="3.2.2" data-path="xds-inp.html"><a href="xds-inp.html#molecular-replacement"><i class="fa fa-check"></i><b>3.2.2</b> Molecular replacement</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="xds-inp.html"><a href="xds-inp.html#alpha-trace-c"><i class="fa fa-check"></i><b>3.3</b> alpha trace C</a><ul>
<li class="chapter" data-level="3.3.1" data-path="xds-inp.html"><a href="xds-inp.html#primary-structure-peptide-bond"><i class="fa fa-check"></i><b>3.3.1</b> Primary structure peptide bond</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="xds-inp.html"><a href="xds-inp.html#preguntas-a-responder"><i class="fa fa-check"></i><b>3.4</b> Preguntas a responder</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="literature.html"><a href="literature.html"><i class="fa fa-check"></i><b>4</b> Literature</a></li>
<li class="chapter" data-level="5" data-path="pymol.html"><a href="pymol.html"><i class="fa fa-check"></i><b>5</b> Pymol</a><ul>
<li class="chapter" data-level="5.1" data-path="pymol.html"><a href="pymol.html#uniprot"><i class="fa fa-check"></i><b>5.1</b> Uniprot</a></li>
<li class="chapter" data-level="5.2" data-path="pymol.html"><a href="pymol.html#subir-tu-proteína-a-pdb"><i class="fa fa-check"></i><b>5.2</b> subir tu proteína a PDB</a></li>
<li class="chapter" data-level="5.3" data-path="pymol.html"><a href="pymol.html#estructuras"><i class="fa fa-check"></i><b>5.3</b> Estructuras</a></li>
<li class="chapter" data-level="5.4" data-path="pymol.html"><a href="pymol.html#pymol-by-schrodinger"><i class="fa fa-check"></i><b>5.4</b> Pymol by Schrodinger</a><ul>
<li class="chapter" data-level="5.4.1" data-path="pymol.html"><a href="pymol.html#pdb-structrue-file"><i class="fa fa-check"></i><b>5.4.1</b> PDB structrue file</a></li>
<li class="chapter" data-level="5.4.2" data-path="pymol.html"><a href="pymol.html#interactive-command-line"><i class="fa fa-check"></i><b>5.4.2</b> Interactive command line</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="pymol.html"><a href="pymol.html#sars-cov-2"><i class="fa fa-check"></i><b>5.5</b> SARS-Cov-2</a><ul>
<li class="chapter" data-level="5.5.1" data-path="pymol.html"><a href="pymol.html#s-protein-spike"><i class="fa fa-check"></i><b>5.5.1</b> S protein (Spike)</a></li>
<li class="chapter" data-level="5.5.2" data-path="pymol.html"><a href="pymol.html#paper"><i class="fa fa-check"></i><b>5.5.2</b> Paper</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="pymol.html"><a href="pymol.html#figure-scripting-by-example-interactive-automation-with-hotkeys-pymolrc-startup-script"><i class="fa fa-check"></i><b>5.6</b> Figure scripting by example Interactive Automation with hotkeys pymolrc startup script</a></li>
<li class="chapter" data-level="5.7" data-path="pymol.html"><a href="pymol.html#access-and-modify-atom-data-with-iterate-and-alter-write-pymol-extensions"><i class="fa fa-check"></i><b>5.7</b> Access and modify atom data with iterate and alter write PyMOL extensions</a></li>
<li class="chapter" data-level="5.8" data-path="pymol.html"><a href="pymol.html#anaconda-cloud-with-pymol-2.0"><i class="fa fa-check"></i><b>5.8</b> Anaconda Cloud with PyMOL 2.0</a></li>
<li class="chapter" data-level="5.9" data-path="pymol.html"><a href="pymol.html#exercises"><i class="fa fa-check"></i><b>5.9</b> Exercises</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="secuencias.html"><a href="secuencias.html"><i class="fa fa-check"></i><b>6</b> Secuencias</a><ul>
<li class="chapter" data-level="6.1" data-path="secuencias.html"><a href="secuencias.html#the-protein-side-of-the-central-dogma-permanence-and-change-thermoynamic-theory-of-protein-folding"><i class="fa fa-check"></i><b>6.1</b> The protein side of the central dogma: Permanence and change thermoynamic theory of protein folding</a></li>
<li class="chapter" data-level="6.2" data-path="secuencias.html"><a href="secuencias.html#in-vitro-models"><i class="fa fa-check"></i><b>6.2</b> In vitro Models</a></li>
<li class="chapter" data-level="6.3" data-path="secuencias.html"><a href="secuencias.html#gibbs-free-energy-explained"><i class="fa fa-check"></i><b>6.3</b> Gibbs free Energy Explained</a></li>
<li class="chapter" data-level="6.4" data-path="secuencias.html"><a href="secuencias.html#protein-folding-kinetics"><i class="fa fa-check"></i><b>6.4</b> Protein Folding Kinetics</a></li>
<li class="chapter" data-level="6.5" data-path="secuencias.html"><a href="secuencias.html#free-energy-of-folding-for-taq"><i class="fa fa-check"></i><b>6.5</b> Free energy of folding for Taq</a></li>
<li class="chapter" data-level="6.6" data-path="secuencias.html"><a href="secuencias.html#thermodynamics"><i class="fa fa-check"></i><b>6.6</b> Thermodynamics</a></li>
<li class="chapter" data-level="6.7" data-path="secuencias.html"><a href="secuencias.html#phi-value-analysis"><i class="fa fa-check"></i><b>6.7</b> Phi Value analysis</a></li>
<li class="chapter" data-level="6.8" data-path="secuencias.html"><a href="secuencias.html#ejercicio-real"><i class="fa fa-check"></i><b>6.8</b> Ejercicio Real</a></li>
<li class="chapter" data-level="6.9" data-path="secuencias.html"><a href="secuencias.html#genealogy"><i class="fa fa-check"></i><b>6.9</b> Genealogy</a><ul>
<li class="chapter" data-level="6.9.1" data-path="secuencias.html"><a href="secuencias.html#comprehensive-databases"><i class="fa fa-check"></i><b>6.9.1</b> comprehensive databases</a></li>
<li class="chapter" data-level="6.9.2" data-path="secuencias.html"><a href="secuencias.html#other-databases"><i class="fa fa-check"></i><b>6.9.2</b> other databases</a></li>
</ul></li>
<li class="chapter" data-level="6.10" data-path="secuencias.html"><a href="secuencias.html#chemical-methods-and-thermal-denaturation-estimation-of-thermodynamic-stability-disordered-proteins"><i class="fa fa-check"></i><b>6.10</b> Chemical methods and thermal denaturation Estimation of thermodynamic stability Disordered proteins</a></li>
<li class="chapter" data-level="6.11" data-path="secuencias.html"><a href="secuencias.html#the-ribosome"><i class="fa fa-check"></i><b>6.11</b> The ribosome</a></li>
<li class="chapter" data-level="6.12" data-path="secuencias.html"><a href="secuencias.html#evolution-of-the-ribosome"><i class="fa fa-check"></i><b>6.12</b> Evolution of the ribosome</a></li>
<li class="chapter" data-level="6.13" data-path="secuencias.html"><a href="secuencias.html#interactive-exercises-protein-folding-and-the-ribosome."><i class="fa fa-check"></i><b>6.13</b> Interactive exercises protein folding and the ribosome.</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="applications.html"><a href="applications.html"><i class="fa fa-check"></i><b>7</b> Applications</a></li>
<li class="chapter" data-level="8" data-path="final-words.html"><a href="final-words.html"><i class="fa fa-check"></i><b>8</b> Final Words</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Structural Bioinformatics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="pymol" class="section level1">
<h1><span class="header-section-number">Chapter 5</span> Pymol</h1>
<div id="uniprot" class="section level2">
<h2><span class="header-section-number">5.1</span> Uniprot</h2>
<p>Swiss-prot
Proteínas caracterizadas bioquímicamente
Protein Data Bank</p>
<p>Rfree de la proteína debe estar bien</p>
<p><strong>R-free</strong>
Es una medida de la calidad del modelo de la proteína en función de qué tan bien explica los datos experimentales</p>
<ul>
<li>Lo que hacemos con un pdb file
-estamos generando un modelo que explica las reflecciones de un cristal</li>
<li>.1 es muy bueno, entre más bajo es mejor
El 25%</li>
<li>sí tienes una resolución muy baja eso te permite diferenciar mucho los atomos por lo que esperas un R-free muy bajo</li>
<li>una resolución de menos de un armstrong te debe dar menos resolución</li>
<li>R-free en una estructura</li>
</ul>
</div>
<div id="subir-tu-proteína-a-pdb" class="section level2">
<h2><span class="header-section-number">5.2</span> subir tu proteína a PDB</h2>
<ul>
<li>subir el file con las reflections</li>
<li>datos del cristal</li>
<li>contenido del solvente</li>
</ul>
</div>
<div id="estructuras" class="section level2">
<h2><span class="header-section-number">5.3</span> Estructuras</h2>
<p><a href="http://pfam.xfam.org">dominios en pfam</a>
<strong>Dominio</strong>
una cadena
<strong>Cuaternario</strong>
Hablamos de más de una cadena
** Estructura super secundaria<strong>
</strong>Plegamiento de proteínas**
-
- 1000 y 2000
- beta trefoil
- jelly-roll
- immunoglobulin fold
- TIM barrel
- ferrodoxin fold
- updown bundle
- muchas distintas secuencias pueden darte el mismo plegamiento</p>
<p>Genomas predecimos proteinas, se guardan en uniprot, se resuelven las estructuras y se hace PDB, luego deep<br />
Subset del PDB en donde se clasifican las proteínas por origen evolutivo común
deep mind y alfa fold en google.</p>
<p><a href="http://scop.mrc-lmb.cam.ac.uk/">scop</a></p>
<ul>
<li>dos proteínas pertencen al mismo plegamiento si contienen dos estructuras en la misma conformación</li>
</ul>
</div>
<div id="pymol-by-schrodinger" class="section level2">
<h2><span class="header-section-number">5.4</span> Pymol by Schrodinger</h2>
<p><a href="https://swift.cmbi.umcn.nl/gv/dssp/">dssp descripción geométrica de los atomos</a></p>
<p><a href="https://pymolwiki.org/index.php/Main_Page">pymol wiki</a></p>
<p><a href="http://molprobity.biochem.duke.edu">mol probity</a>
Pudes desarrollar un metodo para ver clash con pymol y puntos</p>
<p>Dssp indica las coordenadas para la helix, beta, etc.
muchos pdbs tienen secondary structure definition by the program DSSP</p>
<ul>
<li>Model</li>
<li>roving density</li>
<li>ray tracing
<ul>
<li>rendering image</li>
<li>subir una imagen a artículo</li>
<li><code>ray</code>en la linea de comando superior</li>
</ul></li>
</ul>
<div id="pdb-structrue-file" class="section level3">
<h3><span class="header-section-number">5.4.1</span> PDB structrue file</h3>
<p>-header
- datos del deposito
- si fue artificial
- enzyme clasification 3.1.1.7 en KEGG
- te da la info de lo que es la proteína
- metodo de resolución
- organimso
- autores
- versiones
- datos utilizados por el refinment
- Valores de ramachandran
- si la estructura fue revisada
- secuencia en el código de tres letras</p>
<p><strong>¿Qué hago si no encuentro mi proteína en pdb?</strong>
si no encuentro mi proteína en pdb puedeo en swissmodel.expasy.org puedo generar modelos de homología en subiendo mi secuencia comparandola con secuencias que tienen estructura y en función de eso generar un modelo de homología. Si no entonces puedo ir al software robetta de david baker que permite predecir estructuras ab initio utilizando estructura química y no evolución. Utiliza unas librerías de fragmentos. Deep minda con su algoritmo alpha-fold hace algo parecido.</p>
<p>coot lo usamos cuando queremos trabajar en un modelo no terminado, es más una herramienta cristalográfica mientras que pymol es más de visualización.</p>
<p>heath es del solvente</p>
</div>
<div id="interactive-command-line" class="section level3">
<h3><span class="header-section-number">5.4.2</span> Interactive command line</h3>
<pre><code># cargar la molécula
fetch 1acj
# file--export--save molecule--save as .pdb
# borra todo 
delete all

# hide cartoon 
# show lines 
# color by element 
# A action preset--ligandsites--cartoon 



# las cruzes son aguas

# regresar a la conformacion original
reset

bbplane 
run /path/bbplane.py

bfactor putty</code></pre>
</div>
</div>
<div id="sars-cov-2" class="section level2">
<h2><span class="header-section-number">5.5</span> SARS-Cov-2</h2>
<div id="s-protein-spike" class="section level3">
<h3><span class="header-section-number">5.5.1</span> S protein (Spike)</h3>
<p>Solved by cryogenic electon microscopy</p>
<ul>
<li>cilindrical shape</li>
<li>heavyly glycosylated</li>
<li>beta sheets where it fuses to the cell</li>
<li>3 identical subunidades</li>
<li>receptor binding domain RBD
<ul>
<li>where it binds to the host cell</li>
<li>2 are in a down conformation</li>
<li>1 is filliped up</li>
</ul></li>
<li>proteases can cut</li>
<li>ACE2 where it fuses</li>
<li>RBD and ACE2
<ul>
<li>2 tyrosine residues</li>
<li>asn</li>
<li>glu and lys 31
<ul>
<li>amid of a glutamin side chain</li>
<li>hydrogen bonding
<ul>
<li>polar and charge residues
<ul>
<li>glu37 - arg 393 +</li>
</ul></li>
</ul></li>
<li>RBD</li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="paper" class="section level3">
<h3><span class="header-section-number">5.5.2</span> Paper</h3>
<ul>
<li>Existen variantes</li>
<li>Utilizaron el</li>
</ul>
<p>Actividad hacer la figura 2 del paper Structural details at the interface between the SARS-CoV-2 RBM and ACE2</p>
<ol style="list-style-type: decimal">
<li>Buscar los PDBs de las figuras</li>
</ol>
<pre><code># primera estructura SARS coronavirus spike receptor-binding domain complexed with its recpetor
fetch 2ajf
# Segunda estructura
fetch 6vw1
set_name 2ajf, 2ajf-sars-cov
set_name 6vw1-targetm 6w1-target-sars-cov-2

# display---sequence 
# display--sequence mode --chain identifiers
# C orange el 2ajf-sars-cov
# C by element

# Cadena A es el receptor 
#
set_name sele,human-receptor-ACE2-sars-cov
# En la etiqueta selection A action remove atoms 
solo quedarnos con A E C
des SARS-CoV y SARS-CoV-2

helical bundle es el receptor humano 
el que tiene más betas es la proteina viral 

AC menos la A de otro color color by 
# Action en un sars y align molecues

# selecting residues 
seleccionar el loop del humano y sars1

set_
interface L residues

# Solo me quedo con las cadenas Es de cov2
2e e 

target es sars-cov1

query es sars-cov-2 

https://www.ebi.ac.uk/msd-srv/ssm/

entre q se acerca más a 1 es mejor 
RMSD 

### Seleccion hecha
- click secuendario en la seleccion actions, find polar contacts, to any atoms, 
  - actions, find , any contacts between chaisn between 4 armstrongs 
  - 

install protego 

</code></pre>
</div>
</div>
<div id="figure-scripting-by-example-interactive-automation-with-hotkeys-pymolrc-startup-script" class="section level2">
<h2><span class="header-section-number">5.6</span> Figure scripting by example Interactive Automation with hotkeys pymolrc startup script</h2>
</div>
<div id="access-and-modify-atom-data-with-iterate-and-alter-write-pymol-extensions" class="section level2">
<h2><span class="header-section-number">5.7</span> Access and modify atom data with iterate and alter write PyMOL extensions</h2>
</div>
<div id="anaconda-cloud-with-pymol-2.0" class="section level2">
<h2><span class="header-section-number">5.8</span> Anaconda Cloud with PyMOL 2.0</h2>
</div>
<div id="exercises" class="section level2">
<h2><span class="header-section-number">5.9</span> Exercises</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="literature.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="secuencias.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["BEII_III_Structural_Bioinfo.pdf", "BEII_III_Structural_Bioinfo.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
