<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 XDS.INP | Structural Bioinformatics</title>
  <meta name="description" content="This is a book for the notes and activities of the structural bioinformatics module of the Bioinformatics and Statistics course." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 XDS.INP | Structural Bioinformatics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a book for the notes and activities of the structural bioinformatics module of the Bioinformatics and Statistics course." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 XDS.INP | Structural Bioinformatics" />
  
  <meta name="twitter:description" content="This is a book for the notes and activities of the structural bioinformatics module of the Bioinformatics and Statistics course." />
  

<meta name="author" content="Alfredo Varela, Paulina Rosales" />


<meta name="date" content="2021-03-12" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="cs-of-covalent-bondas.html"/>
<link rel="next" href="literature.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Bioinformática y Estadística II-III</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Prerequisites</a></li>
<li class="chapter" data-level="2" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html"><i class="fa fa-check"></i><b>2</b> 3 Cs of covalent bondas</a><ul>
<li class="chapter" data-level="2.1" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#shape-of-biological-molecules"><i class="fa fa-check"></i><b>2.1</b> Shape of biological molecules</a></li>
<li class="chapter" data-level="2.2" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#recognition-and-function-of-biological-molecules"><i class="fa fa-check"></i><b>2.2</b> Recognition and function of biological molecules</a></li>
<li class="chapter" data-level="2.3" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#macromolecular-building-blocks"><i class="fa fa-check"></i><b>2.3</b> Macromolecular Building Blocks</a></li>
<li class="chapter" data-level="2.4" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#video-dna"><i class="fa fa-check"></i><b>2.4</b> video DNA</a></li>
<li class="chapter" data-level="2.5" data-path="cs-of-covalent-bondas.html"><a href="cs-of-covalent-bondas.html#evolutivamente"><i class="fa fa-check"></i><b>2.5</b> Evolutivamente</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="xds-inp.html"><a href="xds-inp.html"><i class="fa fa-check"></i><b>3</b> XDS.INP</a><ul>
<li class="chapter" data-level="3.1" data-path="xds-inp.html"><a href="xds-inp.html#lyzosime"><i class="fa fa-check"></i><b>3.1</b> Lyzosime</a></li>
<li class="chapter" data-level="3.2" data-path="xds-inp.html"><a href="xds-inp.html#life-is-the-result-of-the-chemical-activity-of-proteins"><i class="fa fa-check"></i><b>3.2</b> Life is the result of the chemical activity of proteins</a><ul>
<li class="chapter" data-level="3.2.1" data-path="xds-inp.html"><a href="xds-inp.html#determinación-de-las-bases-con-molecular-replacement"><i class="fa fa-check"></i><b>3.2.1</b> Determinación de las bases con molecular replacement</a></li>
<li class="chapter" data-level="3.2.2" data-path="xds-inp.html"><a href="xds-inp.html#molecular-replacement"><i class="fa fa-check"></i><b>3.2.2</b> Molecular replacement</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="xds-inp.html"><a href="xds-inp.html#alpha-trace-c"><i class="fa fa-check"></i><b>3.3</b> alpha trace C</a><ul>
<li class="chapter" data-level="3.3.1" data-path="xds-inp.html"><a href="xds-inp.html#primary-structure-peptide-bond"><i class="fa fa-check"></i><b>3.3.1</b> Primary structure peptide bond</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="xds-inp.html"><a href="xds-inp.html#preguntas-a-responder"><i class="fa fa-check"></i><b>3.4</b> Preguntas a responder</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="literature.html"><a href="literature.html"><i class="fa fa-check"></i><b>4</b> Literature</a></li>
<li class="chapter" data-level="5" data-path="pymol.html"><a href="pymol.html"><i class="fa fa-check"></i><b>5</b> Pymol</a><ul>
<li class="chapter" data-level="5.1" data-path="pymol.html"><a href="pymol.html#uniprot"><i class="fa fa-check"></i><b>5.1</b> Uniprot</a></li>
<li class="chapter" data-level="5.2" data-path="pymol.html"><a href="pymol.html#subir-tu-proteína-a-pdb"><i class="fa fa-check"></i><b>5.2</b> subir tu proteína a PDB</a></li>
<li class="chapter" data-level="5.3" data-path="pymol.html"><a href="pymol.html#estructuras"><i class="fa fa-check"></i><b>5.3</b> Estructuras</a></li>
<li class="chapter" data-level="5.4" data-path="pymol.html"><a href="pymol.html#pymol-by-schrodinger"><i class="fa fa-check"></i><b>5.4</b> Pymol by Schrodinger</a><ul>
<li class="chapter" data-level="5.4.1" data-path="pymol.html"><a href="pymol.html#pdb-structrue-file"><i class="fa fa-check"></i><b>5.4.1</b> PDB structrue file</a></li>
<li class="chapter" data-level="5.4.2" data-path="pymol.html"><a href="pymol.html#interactive-command-line"><i class="fa fa-check"></i><b>5.4.2</b> Interactive command line</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="pymol.html"><a href="pymol.html#sars-cov-2"><i class="fa fa-check"></i><b>5.5</b> SARS-Cov-2</a><ul>
<li class="chapter" data-level="5.5.1" data-path="pymol.html"><a href="pymol.html#s-protein-spike"><i class="fa fa-check"></i><b>5.5.1</b> S protein (Spike)</a></li>
<li class="chapter" data-level="5.5.2" data-path="pymol.html"><a href="pymol.html#paper"><i class="fa fa-check"></i><b>5.5.2</b> Paper</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="pymol.html"><a href="pymol.html#figure-scripting-by-example-interactive-automation-with-hotkeys-pymolrc-startup-script"><i class="fa fa-check"></i><b>5.6</b> Figure scripting by example Interactive Automation with hotkeys pymolrc startup script</a></li>
<li class="chapter" data-level="5.7" data-path="pymol.html"><a href="pymol.html#access-and-modify-atom-data-with-iterate-and-alter-write-pymol-extensions"><i class="fa fa-check"></i><b>5.7</b> Access and modify atom data with iterate and alter write PyMOL extensions</a></li>
<li class="chapter" data-level="5.8" data-path="pymol.html"><a href="pymol.html#anaconda-cloud-with-pymol-2.0"><i class="fa fa-check"></i><b>5.8</b> Anaconda Cloud with PyMOL 2.0</a></li>
<li class="chapter" data-level="5.9" data-path="pymol.html"><a href="pymol.html#exercises"><i class="fa fa-check"></i><b>5.9</b> Exercises</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="secuencias.html"><a href="secuencias.html"><i class="fa fa-check"></i><b>6</b> Secuencias</a><ul>
<li class="chapter" data-level="6.1" data-path="secuencias.html"><a href="secuencias.html#the-protein-side-of-the-central-dogma-permanence-and-change-thermoynamic-theory-of-protein-folding"><i class="fa fa-check"></i><b>6.1</b> The protein side of the central dogma: Permanence and change thermoynamic theory of protein folding</a></li>
<li class="chapter" data-level="6.2" data-path="secuencias.html"><a href="secuencias.html#in-vitro-models"><i class="fa fa-check"></i><b>6.2</b> In vitro Models</a></li>
<li class="chapter" data-level="6.3" data-path="secuencias.html"><a href="secuencias.html#gibbs-free-energy-explained"><i class="fa fa-check"></i><b>6.3</b> Gibbs free Energy Explained</a></li>
<li class="chapter" data-level="6.4" data-path="secuencias.html"><a href="secuencias.html#protein-folding-kinetics"><i class="fa fa-check"></i><b>6.4</b> Protein Folding Kinetics</a></li>
<li class="chapter" data-level="6.5" data-path="secuencias.html"><a href="secuencias.html#free-energy-of-folding-for-taq"><i class="fa fa-check"></i><b>6.5</b> Free energy of folding for Taq</a></li>
<li class="chapter" data-level="6.6" data-path="secuencias.html"><a href="secuencias.html#thermodynamics"><i class="fa fa-check"></i><b>6.6</b> Thermodynamics</a></li>
<li class="chapter" data-level="6.7" data-path="secuencias.html"><a href="secuencias.html#phi-value-analysis"><i class="fa fa-check"></i><b>6.7</b> Phi Value analysis</a></li>
<li class="chapter" data-level="6.8" data-path="secuencias.html"><a href="secuencias.html#ejercicio-real"><i class="fa fa-check"></i><b>6.8</b> Ejercicio Real</a></li>
<li class="chapter" data-level="6.9" data-path="secuencias.html"><a href="secuencias.html#genealogy"><i class="fa fa-check"></i><b>6.9</b> Genealogy</a><ul>
<li class="chapter" data-level="6.9.1" data-path="secuencias.html"><a href="secuencias.html#comprehensive-databases"><i class="fa fa-check"></i><b>6.9.1</b> comprehensive databases</a></li>
<li class="chapter" data-level="6.9.2" data-path="secuencias.html"><a href="secuencias.html#other-databases"><i class="fa fa-check"></i><b>6.9.2</b> other databases</a></li>
</ul></li>
<li class="chapter" data-level="6.10" data-path="secuencias.html"><a href="secuencias.html#chemical-methods-and-thermal-denaturation-estimation-of-thermodynamic-stability-disordered-proteins"><i class="fa fa-check"></i><b>6.10</b> Chemical methods and thermal denaturation Estimation of thermodynamic stability Disordered proteins</a></li>
<li class="chapter" data-level="6.11" data-path="secuencias.html"><a href="secuencias.html#the-ribosome"><i class="fa fa-check"></i><b>6.11</b> The ribosome</a></li>
<li class="chapter" data-level="6.12" data-path="secuencias.html"><a href="secuencias.html#evolution-of-the-ribosome"><i class="fa fa-check"></i><b>6.12</b> Evolution of the ribosome</a></li>
<li class="chapter" data-level="6.13" data-path="secuencias.html"><a href="secuencias.html#interactive-exercises-protein-folding-and-the-ribosome."><i class="fa fa-check"></i><b>6.13</b> Interactive exercises protein folding and the ribosome.</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html"><i class="fa fa-check"></i><b>7</b> Gunnar von Heijne semminar</a><ul>
<li class="chapter" data-level="7.1" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#the-signal-sequence"><i class="fa fa-check"></i><b>7.1</b> The signal sequence</a></li>
<li class="chapter" data-level="7.2" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#can-someone-identify-such-sequences-with-bioinforatics-on-data-bases"><i class="fa fa-check"></i><b>7.2</b> Can someone identify such sequences with bioinforatics on data bases</a></li>
<li class="chapter" data-level="7.3" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#machine-learning-approaches"><i class="fa fa-check"></i><b>7.3</b> Machine learning approaches</a><ul>
<li class="chapter" data-level="7.3.1" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#you-need"><i class="fa fa-check"></i><b>7.3.1</b> You need</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#memebrane-proteins"><i class="fa fa-check"></i><b>7.4</b> Memebrane proteins</a></li>
<li class="chapter" data-level="7.5" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#topology-prediction"><i class="fa fa-check"></i><b>7.5</b> Topology prediction</a><ul>
<li class="chapter" data-level="7.5.1" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#toppred"><i class="fa fa-check"></i><b>7.5.1</b> TopPred</a></li>
<li class="chapter" data-level="7.5.2" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#tmhmm"><i class="fa fa-check"></i><b>7.5.2</b> TMHMM</a></li>
<li class="chapter" data-level="7.5.3" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#topcons"><i class="fa fa-check"></i><b>7.5.3</b> TOPCONS</a></li>
<li class="chapter" data-level="7.5.4" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#how-good-are-topology-predictors"><i class="fa fa-check"></i><b>7.5.4</b> How good are topology predictors</a></li>
<li class="chapter" data-level="7.5.5" data-path="gunnar-von-heijne-semminar.html"><a href="gunnar-von-heijne-semminar.html#d-structure-of-membrane-proteinns-from-genomic-sequencing"><i class="fa fa-check"></i><b>7.5.5</b> 3D structure of membrane proteinns from Genomic Sequencing</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html"><i class="fa fa-check"></i><b>8</b> Cotranslational protein folding: the when and why of che</a><ul>
<li class="chapter" data-level="8.1" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#defects-in-protein-folding-lead-to-human-disease"><i class="fa fa-check"></i><b>8.1</b> Defects in protein folding lead to human disease</a></li>
<li class="chapter" data-level="8.2" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#molecular-chaperone-socntrol-every-aspect-f-cellular-protein-homeostasis"><i class="fa fa-check"></i><b>8.2</b> Molecular chaperone socntrol every aspect f cellular protein homeostasis</a></li>
<li class="chapter" data-level="8.3" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#clip-chaperones-linked-to-protein-synthesis-folding-of-newly-made-proteins"><i class="fa fa-check"></i><b>8.3</b> CLIP chaperones linked to protein synthesis folding of newly made proteins</a></li>
<li class="chapter" data-level="8.4" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#co-translation-folding-the-vectorial-nature-of-translation-determines-the-folding-pathway"><i class="fa fa-check"></i><b>8.4</b> Co-translation folding: the vectorial nature of translation determines the folding pathway</a></li>
<li class="chapter" data-level="8.5" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#biogenesis-begins-at-the-ribosome"><i class="fa fa-check"></i><b>8.5</b> Biogenesis begins at the ribosome</a><ul>
<li class="chapter" data-level="8.5.1" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#interaction-of-nascent-chains-with-hsp70"><i class="fa fa-check"></i><b>8.5.1</b> interaction of nascent chains with HSP70</a></li>
</ul></li>
<li class="chapter" data-level="8.6" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#how-do-hsp70-and-tric-engage-their-cognate-substrates-co-translationally"><i class="fa fa-check"></i><b>8.6</b> How do Hsp70 and TRiC engage their cognate substrates co-translationally?</a></li>
<li class="chapter" data-level="8.7" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#what-detemines-binding-kinetics-of-elongation-rate"><i class="fa fa-check"></i><b>8.7</b> what detemines binding: kinetics of elongation rate?</a></li>
<li class="chapter" data-level="8.8" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#why-we-age"><i class="fa fa-check"></i><b>8.8</b> ## why we age</a></li>
<li class="chapter" data-level="8.9" data-path="cotranslational-protein-folding-the-when-and-why-of-che.html"><a href="cotranslational-protein-folding-the-when-and-why-of-che.html#practice"><i class="fa fa-check"></i><b>8.9</b> Practice</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="protein-design.html"><a href="protein-design.html"><i class="fa fa-check"></i><b>9</b> Protein design</a><ul>
<li class="chapter" data-level="9.1" data-path="protein-design.html"><a href="protein-design.html#power-of-evolution"><i class="fa fa-check"></i><b>9.1</b> Power of evolution</a></li>
<li class="chapter" data-level="9.2" data-path="protein-design.html"><a href="protein-design.html#native-structures-are-conformational-free-energy-minima"><i class="fa fa-check"></i><b>9.2</b> Native structures are conformational free energy minima</a></li>
<li class="chapter" data-level="9.3" data-path="protein-design.html"><a href="protein-design.html#why-bother"><i class="fa fa-check"></i><b>9.3</b> Why bother?</a></li>
<li class="chapter" data-level="9.4" data-path="protein-design.html"><a href="protein-design.html#protein-engineering-and-design-approaches"><i class="fa fa-check"></i><b>9.4</b> Protein engineering and design approaches</a><ul>
<li class="chapter" data-level="9.4.1" data-path="protein-design.html"><a href="protein-design.html#summary"><i class="fa fa-check"></i><b>9.4.1</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="protein-design.html"><a href="protein-design.html#fragment-based-design"><i class="fa fa-check"></i><b>9.5</b> Fragment-based design</a><ul>
<li class="chapter" data-level="9.5.1" data-path="protein-design.html"><a href="protein-design.html#how-do-all-folds-compare"><i class="fa fa-check"></i><b>9.5.1</b> How do all folds compare</a></li>
<li class="chapter" data-level="9.5.2" data-path="protein-design.html"><a href="protein-design.html#summary-iii"><i class="fa fa-check"></i><b>9.5.2</b> summary III</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Structural Bioinformatics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="xds.inp" class="section level1">
<h1><span class="header-section-number">Chapter 3</span> XDS.INP</h1>
<div class="sourceCode" id="cb1"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb1-1"><a href="xds-inp.html#cb1-1"></a><span class="fu">scp</span> XDS.INP avarela@tepeu.lcg.unam.mx:/export/storage/users/avarela</span>
<span id="cb1-2"><a href="xds-inp.html#cb1-2"></a></span>
<span id="cb1-3"><a href="xds-inp.html#cb1-3"></a><span class="co"># Ejecutar la lectura </span></span>
<span id="cb1-4"><a href="xds-inp.html#cb1-4"></a><span class="ex">xds_par</span> </span></code></pre></div>
<pre><code>UNIT_CELL_CONSTANTs= 38.9380 377.0
JOB
x-ray_wavelength = longitud de onda  al que le disparamos 
- datos de donde a 
- detector parametro para que no se sobrecargue 
- exlclye la resolución tal porque hay un aro de hielo </code></pre>
<p>Protein cristal fishing
- pescas el cristal y lo metes en nitrógeno líquido
- ice ring diffraction pattern se ve como un aro al disparar
- tienes que excluirla de tu análisis
- solo trabajas con el cristal si tiene hielo si no tienes de otra
-</p>
<ul>
<li>el grupo espacial entre más alto sea tiene mayor simetría</li>
</ul>
<p>Se expresan las proteínas, se purifican se cristalizan.</p>
<ul>
<li>La celda unitaria
<ul>
<li>unidad minima definida en xy y z que se necesita para reconsturir el cristal completo.
<ul>
<li>si lo trasladas y lo rotas
<ul>
<li>unidad asimétrica es mas chiva que la celda unitaria</li>
<li>procesos de rotación y traslacion se genera el cristal entero</li>
</ul></li>
</ul></li>
</ul></li>
<li>PSI Paul scharrer institut
<ul>
<li>Se generan los rayos X, se le disparan a los cristales de proteina acomodados de forma ordenada
<ul>
<li>patrones de difracción se ven como puntitos
<ul>
<li>se puede extraer el patrón que corresponde a las fases o amplitudes de esas ondas
<ul>
<li>se puede determinar la d</li>
</ul></li>
</ul></li>
</ul></li>
<li>PX-2</li>
</ul></li>
</ul>
<div id="lyzosime" class="section level2">
<h2><span class="header-section-number">3.1</span> Lyzosime</h2>
</div>
<div id="life-is-the-result-of-the-chemical-activity-of-proteins" class="section level2">
<h2><span class="header-section-number">3.2</span> Life is the result of the chemical activity of proteins</h2>
<ul>
<li>spacefilling Representation
<ul>
<li>radio de vanderballs es toda la probilidad de encontrar girando a las moléculas
<ul>
<li>macromolecular crowding</li>
<li>medir cosas con volumen</li>
<li></li>
</ul></li>
</ul></li>
</ul>
<p>space group es la constante unitaria
- reflections on file</p>
<p><a href="https://www.biologie.uni-konstanz.de/diederichs/">grupo de xds</a></p>
<div id="determinación-de-las-bases-con-molecular-replacement" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Determinación de las bases con molecular replacement</h3>
<p>projecto tutorial-LCG</p>
<p>seuqnce file lyzo.txt</p>
</div>
<div id="molecular-replacement" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Molecular replacement</h3>
<p>Data analysis</p>
<ul>
<li>xtriage
job:title: data analysys
<ul>
<li>xds_ascii.hkl</li>
</ul></li>
</ul>
<p>cuando nosotros le disparamos a un cristal de proteinas el cristal está girando
entre más alejados estemos del centro es más alta la resolución
entre más simetría puedes tener más redundancia
- mejora tu completness
- mosaicicidad es de mala calidad
- redes internas en el cristal
- buena calidad</p>
<p>le decimos que la</p>
<p>asmu le explicamos cuantas moleculas de lizosima tenemos por celda unitaria
- sabemos que hay 1 por cada celda unitaria
- add search templado
- le decimos que use la pdb</p>
</div>
</div>
<div id="alpha-trace-c" class="section level2">
<h2><span class="header-section-number">3.3</span> alpha trace C</h2>
<ul>
<li>carbobo alfa nos referimos al backbone de toda la proteína
<ul>
<li>son el esqueleto de la proteina
<a href="https://swissmodel.expasy.org/course">Principles of protein structure, comparative portein modelling and visualization</a></li>
</ul></li>
<li>peptide bond es en donde se unen dos resiudos</li>
<li>beta sheet se ven com flechas</li>
<li>si queremos seguir el camino de la proteina desde el amino</li>
<li>carbox terminal inicia en rojo</li>
<li>amino terminal es el azul</li>
</ul>
<div id="primary-structure-peptide-bond" class="section level3">
<h3><span class="header-section-number">3.3.1</span> Primary structure peptide bond</h3>
<p>20 aminoacidos naturales
- glicina el más chico de la cadena lateral
- alanina
- valina
- leucina 3</p>
<ul>
<li>aromativos
<ul>
<li>fenilalaninda</li>
</ul></li>
<li>positivas
<ul>
<li>histidina en mcuhas reacciones</li>
</ul></li>
<li>polar sin carga
<ul>
<li>prolina genera distorciones en las alfa helices</li>
</ul></li>
<li>triptofano se usa en fluorescencia</li>
</ul>
<p>LA frecuencia de residuos como una función de su número de codones
- Ramachandran plot</p>
<p><a href="http://bioinformatics.org/molvis/phipsi/">Ramachandran plot</a></p>
<p>Radios de vander balls
clash
- interacción fisicamente imposible</p>
<ul>
<li>angulos phi y pshi
<ul>
<li>descripción de los grados de libertad que existene entre dos atomos</li>
</ul></li>
</ul>
<p>solamente existen ciertas combinaciones de angulos phy y psi permitidos por los grados de libertad
- cuando el phy esta en 0 no hay momento en el que el psi pueda existir sin clash</p>
<p>Se toma empiricamente y se cuantifican</p>
<p>Tarea hacer el practice quiz de ramachandran plot</p>
<p>fenex refine
- usa lo que le dimos</p>
</div>
</div>
<div id="preguntas-a-responder" class="section level2">
<h2><span class="header-section-number">3.4</span> Preguntas a responder</h2>
<p><a href="https://cdn.rcsb.org/pdb101/molecular-machinery/">preguntas a contestar</a></p>
<ol style="list-style-type: decimal">
<li>¿Cuántas moléculas de miosina se necesitan para sostener una pelota de baseball?
2 trillones de moléculas de miosina</li>
</ol>
<p>2.¿Qué residuos forman el cromóforo de GFP?
residuos 65-67 (Ser- dehydro Tyr- Gly) serina, dehidro tirosina y glicina cuyo backbone ciclado forma el anillo imidazolidone. (SYG)</p>
<p>3.¿Cual es uno de los más poderosos inhibidores de la RNA polimerasa?
La alfa-amanitina, un péptido pequeño y circular creado por el hongo <em>Amanita muscaria</em></p>
<ol start="4" style="list-style-type: decimal">
<li><p>Describir la actividad de la DNA helicasa.
Permite el desenrrollamiento y la exposición de las bases para que la polimersa pueda acceder para sintetizar la hebra complementaria de DNA o RNA.</p></li>
<li><p>¿Que es mas grande un ribosoma o un virus?
el tamaño del ribosoma bacteriano es de al rededor de 20nm (200 Armstorngs), el eucariótico entre 25-30 nm (250-300 Armstrongs). Los virus más pequeños como el de la familia Picoronaviridae es de 20-30 nm, los de tamaño más grande como los Paramixoviridae pueden medir entre 150-300nm entonces la respuesta es el virus.</p></li>
<li><p>¿Como funciona la lisozima?
Es una enzima que actua en la pared bacteriana al romer las cadenas de carbohidratos destruyendo su integridad estructural. Lo que provoca que su presión osmótica la haga reventar.</p></li>
</ol>
<p>r-free score que no esta sesgado pues es solo
bonds y angulos es que tan apretada está la proteina
angulos la descripción geométrica de todos los ángulos
- clashscore
Ramachandran favored el 95% están cayendo en lasa regiones permitidas de la gráfica
- Ramachandran outliers:
- alanina con un rotamero poco común
-
phenix.refine</p>
<p>Compare statistics
- lo que queremos tener es un pentagono simétrico
-
puente de hidrogeno son 2.5 armstrons
molecula de hibbs está funcionando como un amortiguador</p>
<p>Tarea 3</p>
<p>corregir la lizosima
- abrirla en coot
- jugar con los rotameros
- resolver los clashes
-tutorial-lcg_bad-atoms.pdb</p>
<ul>
<li>tiene interacciones</li>
</ul>
<p>validation en fenyx te dice qué aminoacidos tienen clash
- clashcore 12 bajarlo como a 5.6
- guardarlo como pdb y mandarlo</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="cs-of-covalent-bondas.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="literature.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["BEII_III_Structural_Bioinfo.pdf", "BEII_III_Structural_Bioinfo.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
